build.sbt

```diff
@@ -3633,7 +3633,7 @@ lazy val `engine-runner` = project
       val NI_MODULES =
         "org.graalvm.nativeimage,org.graalvm.nativeimage.builder,org.graalvm.nativeimage.base,org.graalvm.nativeimage.driver,org.graalvm.nativeimage.librarysupport,org.graalvm.nativeimage.objectfile,org.graalvm.nativeimage.pointsto,com.oracle.graal.graal_enterprise,com.oracle.svm.svm_enterprise"
       val JDK_MODULES =
-        "jdk.localedata,jdk.httpserver,java.naming,java.net.http"
+        "jdk.localedata,jdk.httpserver,java.naming,java.net.http,java.desktop,java.xml.crypto,jdk.xml.dom,java.sql.rowset"
```

Comment:
This will very likely break Espresso support. We had to remove `java.desktop` because of Espresso in #9866

Reply:
> `java.desktop` is required by `org.apache.poi.poi`. The NI failed on `Base_Tests` because of that.

That's scary. Probably another dependency on `java.beans` - we should change upstream to make this dependency optional.

Reply:
As a result of using `java.desktop` the _native image_ build now adds:
```
[info]  /enso.ide/built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/bin/libawt.so (jdk_library)
[info]  /enso.ide/built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/bin/libawt_headless.so (jdk_library)
[info]  /enso.ide/built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/bin/libawt_xawt.so (jdk_library)
[info]  /enso.ide/built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/bin/libfontmanager.so (jdk_library)
[info]  /enso.ide/built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/bin/libjava.so (jdk_library_shim)
[info]  /enso.ide/built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/bin/libjavajpeg.so (jdk_library)
[info]  /built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/bin/libjvm.so (jdk_library_shim)
[info]  /built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/bin/liblcms.so (jdk_library)
```
that's unfortunate.

Reply:
Backed out in 4c46657